tailwind.js

```diff
@@ -7,24 +7,36 @@ module.exports = {
       sm: "640px",
       md: "768px",
       lg: "1024px",
-      xl: "1280px"
+      xl: "1440px"
     },
     colors: {
+      // custom
+      vihrea: "#4c7a61",
+      tummanvihrea: "#3b604c",
+      mustanharmaa: "#333333",
+      tummanharmaa: "#666666",
+      harmaa: "#e3e3e3",
+      vaalenharmaa: "#fafafa",
+      valkoinen: "#ffffff",
+      linkkihover: "#294637",
+      linkkivisited: "#2355b47",
+      bordervari: "#e2e8f0",
+
```

Comment:
This naming might be a problem. Naming seems to assume that bordervari exists only for border colors and linkkivisited and linkkihover colors are for links only. More generic naming could be better way to name colors.